components/telemetry-operator/internal/validation/files_validator.go

```diff
@@ -0,0 +1,59 @@
+package validation
+
+import (
+	"fmt"
+	"strings"
+
+	telemetryv1alpha1 "github.com/kyma-project/kyma/components/telemetry-operator/apis/telemetry/v1alpha1"
+)
+
+//go:generate mockery --name FilesValidator --filename files_validator.go
+type FilesValidator interface {
```

Comment:
Let's move the `FilesValidator` to a second PR. It's not really related to the Loki output.